apis/outlook/mail/code/pkg/printers/mail.go

```diff
@@ -74,3 +82,47 @@ func PrintMessage(msg models.Messageable, detailed bool) error {
 func recipientableToString(r models.Recipientable) string {
 	return fmt.Sprintf("%s (%s)", util.Deref(r.GetEmailAddress().GetName()), util.Deref(r.GetEmailAddress().GetAddress()))
 }
+
+func fixMailFolder(folder *models.MailFolderable) error {
```

Comment:
I would appreciate a more descriptive name for this, or a comment explaining exactly what is being fixed.

Reply:
Yeah I should have named these better lol

Reply:
fixed